scripts/deployment/phase2-assets/collaterals/deploy_sfrax_eth.ts

```diff
@@ -0,0 +1,86 @@
+import fs from 'fs'
+import hre from 'hardhat'
+import { getChainId } from '../../../../common/blockchain-utils'
+import { networkConfig } from '../../../../common/configuration'
+import { bn, fp } from '../../../../common/numbers'
+import { expect } from 'chai'
+import { CollateralStatus } from '../../../../common/constants'
+import {
+  getDeploymentFile,
+  getAssetCollDeploymentFilename,
+  IAssetCollDeployments,
+  getDeploymentFilename,
+  fileExists,
+} from '../../common'
+import { priceTimeout, combinedError } from '../../utils'
+import { SFraxEthCollateral } from '../../../../typechain'
+import { ContractFactory } from 'ethers'
+
+async function main() {
+  // ==== Read Configuration ====
+  const [deployer] = await hre.ethers.getSigners()
+
+  const chainId = await getChainId(hre)
+
+  console.log(`Deploying Collateral to network ${hre.network.name} (${chainId})
+    with burner account: ${deployer.address}`)
+
+  if (!networkConfig[chainId]) {
+    throw new Error(`Missing network configuration for ${hre.network.name}`)
+  }
+
+  // Get phase1 deployment
+  const phase1File = getDeploymentFilename(chainId)
+  if (!fileExists(phase1File)) {
+    throw new Error(`${phase1File} doesn't exist yet. Run phase 1`)
+  }
+  // Check previous step completed
+  const assetCollDeploymentFilename = getAssetCollDeploymentFilename(chainId)
+  const assetCollDeployments = <IAssetCollDeployments>getDeploymentFile(assetCollDeploymentFilename)
+
+  const deployedCollateral: string[] = []
+
+  /********  Deploy SFRAX ETH Collateral - sFraxETH  **************************/
+  let sFraxEthOracleAddress: string = networkConfig[chainId].CURVE_POOL_WETH_FRXETH!
+  const SFraxEthCollateralFactory: ContractFactory = await hre.ethers.getContractFactory(
+    'SFraxEthCollateral'
+  )
+
+  const oracleError = combinedError(fp('0.005'), fp('0.0002')) // 0.5% & 0.02%
+
+  const collateral = <SFraxEthCollateral>await SFraxEthCollateralFactory.connect(deployer).deploy(
+    {
+      priceTimeout: priceTimeout.toString(),
+      chainlinkFeed: networkConfig[chainId].chainlinkFeeds.ETH,
+      oracleError: oracleError.toString(),
+      erc20: networkConfig[chainId].tokens.sfrxETH,
+      maxTradeVolume: fp('1e6').toString(), // $1m,
+      oracleTimeout: '3600', // 1 hr
+      targetName: hre.ethers.utils.formatBytes32String('ETH'),
+      defaultThreshold: fp('0.02').add(oracleError).toString(), // ~2.5%
+      delayUntilDefault: bn('86400').toString(), // 24h
+    },
+    '0', // revenueHiding = 0
```

Comment:
let's use `fp('1e-6')` like we do elsewhere when we aren't 100% sure and want to permit small rounding errors

Reply:
Good catch. In Reth and Lido we are using `fp('1e-4').toString(),` should we do the same?

Reply:
cbETH the same

Reply:
Setting to `1e-4` as agreed offline